tests/ui/drop/lint-if-let-rescope.rs

```diff
@@ -0,0 +1,42 @@
+//@ edition:2024
+//@ compile-flags: -Z validate-mir -Zunstable-options
+//@ run-rustfix
+
+#![deny(if_let_rescope)]
+#![allow(irrefutable_let_patterns)]
+#![feature(if_let_rescope)]
```

Comment:
Suggestion: this feature should be hoisted above the deny.

Reply:
Applied